crates/sui-graphql-rpc/src/types/available_range.rs

```diff
@@ -11,6 +11,9 @@ pub(crate) struct AvailableRange {
 }
 
 // TODO: do both in one query?
+
+/// Range of checkpoints that the RPC has data available for (for data that can
```

Comment:
I think one way or another we want to tie all data to available range for consistency, so the text in parentheses may be redundant)

Reply:
```suggestion
/// Range of checkpoints that the RPC is guaranteed to produce a consistent response for.
```

RPC might be able to serve data outside this range, "AvailableRange" is about consistency.